### DISCUSSION TITLE
commercial license apply to all Emgu libraries?

### DISCUSSION THREAD

**Discussion by ErdnaDt** [emgucv/emgucv#780] [General]
Hi,

Just so I understand correctly ... Does the commercial license apply to all Emgu libraries, such as the Emgu.CV.Bitmap? 

Cheers André

**Comment by emgucv** [comment]
The license for Emgu.CV.Bitmap nuget package can be found here:
https://www.nuget.org/packages/Emgu.CV.Bitmap/4.6.0.5131/License

In short, if you do not own a commercial license, you can use the package under GPL. If you own a commercial license, you can use the package under the commercial license terms instead.
